### PR TITLE
test ui elements

### PR DESCRIPTION
**Pull request by mirnawong1**
---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-test-ui-elements-dbt-labs.vercel.app/docs/introduction

### CODE REVIEW

File: website/docs/docs/introduction.md [hunk 1]
```diff
@@ -9,6 +9,18 @@ pagination_prev: null
...
+
+Navigate to **Account Settings** to change the title and description of this page.
+
+Navigate to Account settings in the side menu.
+
+Find the **Settings** section and click on **Integrations**.
+
+Scroll to AI and select the toggle for OpenAI
+
+Enter your API key and click Save.
```
Comment (mirnawong1) [inline website/docs/docs/introduction.md]:
```suggestion
Enter your API key and click `Save`
```

File: website/docs/docs/introduction.md [hunk 2]
```diff
@@ -9,6 +9,18 @@ pagination_prev: null
...
+
+Navigate to **Account Settings** to change the title and description of this page.
+
+Navigate to Account settings in the side menu.
+
+Find the **Settings** section and click on **Integrations**.
+
+Scroll to AI and select the toggle for OpenAI
+
+Enter your API key and click `Save`
```
Comment (mirnawong1) [inline website/docs/docs/introduction.md:22]: didn't capture those ui elements w backticks.

File: website/docs/docs/introduction.md [hunk 3]
```diff
@@ -9,6 +9,18 @@ pagination_prev: null
 
 dbt compiles and runs your analytics code against your data platform, enabling you and your team to collaborate on a single source of truth for metrics, insights, and business definitions. This single source of truth, combined with the ability to define tests for your data, reduces errors when logic changes, and alerts you when issues arise.
 
+Click Save to update the title and description of this page.
+
+Navigate to **Account Settings** to change the title and description of this page.
+
+Navigate to Account settings in the side menu.
```
Comment (matthewshaver) [inline website/docs/docs/introduction.md]:
```suggestion
## Navigate to Account settings in the side menu.
```

File: website/docs/docs/introduction.md [hunk 4]
```diff
@@ -9,6 +9,18 @@ pagination_prev: null
 
 dbt compiles and runs your analytics code against your data platform, enabling you and your team to collaborate on a single source of truth for metrics, insights, and business definitions. This single source of truth, combined with the ability to define tests for your data, reduces errors when logic changes, and alerts you when issues arise.
 
+Click Save to update the title and description of this page.
+
+Navigate to **Account Settings** to change the title and description of this page.
```
Comment (mirnawong1) [inline website/docs/docs/introduction.md]:
```suggestion
Navigate to Account Settings to change the title and description of this page.
```

**Comment by mirnawong1** [comment]
closing test pr

**Pull request closed without merging**